lib/rubocop/cop/mixin/configurable_enforced_style.rb

```diff
@@ -52,8 +52,8 @@ def detected_style
       def detected_style=(style)
         Formatter::DisabledConfigFormatter.detected_styles[cop_name] = style
 
-        return no_acceptable_style! if style.nil?
-        return no_acceptable_style! if style.empty?
+        no_acceptable_style! && return if style.nil?
```

Comment:
I see what you mean. A bit too much logic for a setter perhaps. There's room for improvement, but I think we can leave it for another PR. I'll see if I can come up with something.